app/assets/i18n/_missing_translations_zh_HK.json

```diff
@@ -5,36 +5,36 @@
   ],
   "settingsTab": {
     "general": {
-      "showInContextMenu": "Show LocalSend in context menu"
+      "showInContextMenu": "喺檔案功能表嘅「傳送到」項目顯示 LocalSend"
```

Comment:
This should be clarified in other languages too, to:
Show LocalSend in the “Send to” file context menu